src/conditionals/programs_with_decision_points/medium/q8/Competition.java

```diff
@@ -0,0 +1,33 @@
+package conditionals.programs_with_decision_points.medium.q8;
+
+/*
+ * Write a program that will ask the user for their placement and print out an "award" based on their placement, 
+ * using the following information: 1st place gets a "Gold Trophy", 2nd place gets a "Silver Trophy", 3rd place 
```

Comment:
Change text to `... print out an award based on it using the following information: `